returnn/tf/util/basic.py

```diff
@@ -1339,6 +1339,33 @@ def __call__(self, shape, dtype=None, partition_info=None):
   return LoadTxtFileInitializer()
 
 
+class GammatoneFilterbankInitializer(init_ops.Initializer):
+  """
+  Initializer for a gammatone filterbank, e.g., to initialize weights of a convolutional layer.
+  """
+
+  def __init__(self, **kwargs):
+    """
+    :param kwargs: kwargs for GammatoneFilterbank
+    """
+    from returnn.util.sig_proc import GammatoneFilterbank
+    self.gammatone_filterbank = GammatoneFilterbank(**kwargs)
+
+  def __call__(self, shape, dtype=None, partition_info=None):
```

Comment:
Is this the right default for `dtype`?
What should be returned for this default? I guess a tensor of type float32, right? This would be wrong in your case.

Reply:
I adapted it from the default dtype in `LoadTxtFileInitializer` but I guess you are right and `dtype=tf.float32` should be used by default.